client/state/data-layer/analytics/adTracking.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	trackCustomAdWordsRemarketingEvent,
+	trackCustomFacebookConversionEvent,
+} from 'lib/analytics/ad-tracking';
+import { SIGNUP_STEPS_SURVEY_SET } from 'state/action-types';
+
+function recordSelectedVertical( _, action, next ) {
+	next( action );
```

Comment:
Shouldn't we be extending the `state/analytics` actions and middleware middleware instead to support these other events? That way we can just add extra analytics meta info to the dispatched `SIGNUP_STEPS_SURVEY_SET`